lib/pacio_test_kit/pfe/single_observation_group.rb

```diff
@@ -17,6 +18,16 @@ class SingleObservationGroup < Inferno::TestGroup
       }
       run_as_group
 
+      test from: :pacio_resource_create,
+           title: 'Server creates correct Observation resource from Observation create interaction',
+           config: {
+             inputs: {
+               resource_inputs: {
+                 name: :single_observation_resource_list,
+                 title: 'Default resource for PFESingleObservation to create on the server'
+               }
+             }
+           }
```

Comment:
only need to add create test to one observation profile. It was already added to the clinical test profile